std/functional.d

```diff
@@ -260,6 +210,134 @@ unittest
     static assert(!is(typeof(binaryFun!FuncObj)));
 }
 
+/**
+Transforms a string representing an expression into a n-ary function.
+$(UL
+    $(LI The number of parameters is given by `argc`. If no explicit
+        `argc` is given, it is inferred to be `paramNames.length`.)
+    $(LI Explicit names for the function parameters may be given as
+        `paramNames`. If more than `argc` names are supplied, the
+        extras will be ignored.)
+    $(LI The letters (`a` through `z`) are used as default names
+        for up to the first 26 function parameters.)
+    $(LI `args[index]` can always be used to access any parameter by index,
+        regardless of whether it has been given a name.)
+
+If `fun` is not a `string`,  `stringLambda` aliases itself to `fun`.
+In such cases, `stringLambda` does $(B not) verify that `fun` accepts
+the specified number of arguments. (Try $(XREF traits, arity) instead.)
+)
```

Comment:
oops, that was a typo. I wanted it separate from the list. I'll fix it and make sure the docs look right.

> (I'm surprised this passed the auto-tester, actually.)

Spoke too soon...